autopush-common/src/reliability.rs

```diff
@@ -566,12 +734,31 @@ mod tests {
             .ignore()
             .cmd("ZADD")
             .arg(EXPIRY)
+            .arg(1)
```

Comment:
I would say same clarifying comment regarding status, since I guess this means expiry?